recipes/dammit/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+gem install crb-blast
```

Comment:
Where will the gem be installed? It is important that it is installed into $PREFIX. If that is ensured, the approach would be ok.